jenkins/Jenkinsfile

```diff
@@ -203,6 +204,105 @@ EOT
             }
           }
         }
+        stage('Windows-Build'){
+          agent{
+            kubernetes {
+              label "cloudify-agent-win-${env.BUILD_NUMBER}"
+              yamlFile 'jenkins/build-pod-windows.yaml'
+            }
+          }
+          environment {
+            TOX_ENV = 'pywin'
+            ERLANG_DOWNLOAD_URL = 'http://erlang.org/download/otp_win64_21.3.exe'
+            ERLANG_INSTALLER_PATH = 'C:\\erlang_install.exe'
+            RABBITMQ_DOWNLOAD_URL = 'https://github.com/rabbitmq/rabbitmq-server/releases/download/v3.8.4/rabbitmq-server-3.8.4.exe'
+            RABBITMQ_INSTALLER_PATH = 'C:\\rabbitmq_install.exe'
+            PYTHON = 'C:\\Python36'
+          }
+          options {
+            checkoutToSubdirectory('cloudify-agent')
+            timeout(time: 60, unit: 'MINUTES')
+            timestamps()
+          }
+          steps{
+            script{
+              container('shell'){
+                powershell """
+                  Write-Host "Uninstalling existing erlang installs"
```

Comment:
This looks like you've just copied the agent build powershell script into here- why not download and run it instead, so that we don't have to make any fixes in both places (but by keeping the script separate it's still easy to run it by hand)?

Reply:
OK, just discussed elsewhere and this was taken from the appveyor config which won't be used any more, so no objection.